database/migrations/2024_04_09_165945_update_people_table.php

```diff
@@ -18,7 +18,7 @@ public function up(): void
             $table->char('reside_state', 2)->nullable();
             $table->char('grant_detail', 30)->nullable();
```

Comment:
Maybe just make many of the free responses "string" columns? How certain can we be that 30 characters is enough here? If we put something longer than that, we do not get a useful error message (submission just fails silently).